### ISSUE TITLE
[feature request] Show search suggestions under search bar

### ISSUE THREAD

**Issue by Pl4to42** [zombieFox/nightTab#197]
It would be nice to be able to see search suggestions in the same style as most search engine main pages.

**Comment by zombieFox** [comment]
Hi, sorry I'm not sure I follow exactly. How would this work exactly? Do you mean suggested results under the search bar fetched from the selected search engine? 

What will help me is a description of the problem such a feature might be solving please?

**Comment by Pl4to42** [comment]
Yeah that's what I mean. I'm looking for something like how google can to [this](https://external-content.duckduckgo.com/iu/?u=http%3A%2F%2Fi3.kym-cdn.com%2Fphotos%2Fimages%2Foriginal%2F000%2F022%2F588%2Fgoogle_suggest_7.JPG&f=1&nofb=1). 

I think it would be a really useful feature to make this start page more complete as I rely on the search bar to find pages I've searched before and save some time. By adding this, I could sort through bookmarks on one side of the page and get search suggestions on the other side of the page under the search bar.
